Automattic/woocommerce-payments
# Handle when `get_disputes_needing_response()` returns non-array.

Fixes https://github.com/Automattic/woocommerce-payments/issues/6597.

#### Changes proposed in this Pull Request

Handle the case when `WC_Payments_Task_Disputes::get_disputes_needing_response()` returns a non-array.

#### Testing instructions

* Add a `return null;` in the beginning of `WC_Payments_Task_Disputes::get_disputes_needing_response()`.
* Load WC home (/wp-admin/admin.php?page=wc-admin).
* No error and no dispute task should appear under the `Things to do next` section.

-------------------


**Post merge**
